utils/create-bundles/React/bundler.ts

```diff
@@ -0,0 +1,52 @@
+import { build, BuildOptions } from 'esbuild';
+import { existsSync, mkdirSync, removeSync } from 'fs-extra';
+import { join } from 'path';
+import ignoreMissingCss from '../plugins/ignore-missing-css';
+import {
+  getDestinationPathByDemo, getSourcePathByDemo, createDemoLayout,
+} from '../helper';
+import { Bundler } from '../helper/bundler';
+import { Demo, Framework } from '../helper/types';
+
+export default class ReactBundler implements Bundler {
+  framework: Framework = 'React';
+
+  getBuildOptions = (demo: Demo): BuildOptions => {
+    const sourceDemoPath = getSourcePathByDemo(demo, this.framework);
+    const destinationDemoPath = getDestinationPathByDemo(demo, this.framework);
+
+    const options: BuildOptions = {
+      bundle: true,
+      minify: true,
+      loader: { '.js': 'jsx' },
+      entryNames: '[dir]/bundle.[hash]',
+      outdir: destinationDemoPath,
+      entryPoints: this.#getEntryPoints(sourceDemoPath),
+      plugins: [ignoreMissingCss],
+      external: ['*.png', '*.svg'],
+    };
+
+    return options;
+  };
+
+  buildDemo = async (demo: Demo): Promise<void> => {
```

Comment:
this method looks identical for both classes - can we avoid duplicating it?

Reply:
I'm afraid that in the future we will need to make changes to each method separately

Reply:
The general rule of thumb is to write the most efficient and simple code at present, and not make it redundant/non-optimal at present for the sake of probable new circumstances in the future.

Reply:
Okay, I prepare commit with this https://github.com/DevExpress/devextreme-demos/pull/2851/commits/dac512795cea7c2c8a1fb65e438bc062ad58b81f